GovUkDesignSystem/GovUkDesignSystemComponents/PaginationViewModel.cs

```diff
@@ -0,0 +1,119 @@
+﻿using System.Collections.Generic;
+
+namespace GovUkDesignSystem.GovUkDesignSystemComponents;
+
+public class PaginationViewModel
+{
+    /// <summary>
+    /// The array of link objects.
+    /// </summary>
+    public List<PaginationItemViewModel> Items { get; set; }
+
+    /// <summary>
+    /// A link to the previous page, if there is a previous page.
+    /// </summary>
+    public PaginationPreviousViewModel Previous { get; set; }
+    
+    /// <summary>
+    /// A link to the next page, if there is a next page.
+    /// </summary>
+    public PaginationNextViewModel Next { get; set; }
+    
+    /// <summary>
+    /// The label for the navigation landmark that wraps the pagination. Defaults to 'results'.
+    /// </summary>
+    public string LandmarkLabel { get; set; }
+    
+    /// <summary>
+    /// The classes you want to add to the pagination nav parent.
+    /// </summary>
+    public string Classes { get; set; }
+    
+    /// <summary>
+    /// The HTML attributes (for example, data attributes) you want to add to the pagination nav parent.
+    /// </summary>
+    public Dictionary<string, string> Attributes { get; set; }
+}
+
+public class PaginationItemViewModel
+{
+    /// <summary>
+    /// The pagination item text - usually a page number.
+    /// </summary>
+    public string Number { get; set; }
+    
+    /// <summary>
+    /// The visually hidden label (for the pagination item) which will be applied to an aria-label and announced
+    /// by screen readers on the pagination item link. Should include page number.
+    /// </summary>
+    public string VisuallyHiddenText { get; set; }
+    
+    /// <summary>
+    /// <b>Required</b>. The link's URL.
+    /// </summary>
+    public string Href { get; set; }
+    
+    /// <summary>
+    /// Set to true to indicate the current page the user is on.
+    /// </summary>
+    public bool Current { get; set; }
```

Comment:
Having this be set per-item feels like an easy failure point (e.g. accidentally setting multiple items as current). Any thoughts on making be a property of `PaginationViewModel` instead?
I'm fine with it staying like this if changing it will cause headaches.

Reply:
Cool, happy to leave it like this then 🙂